rustler_sys/build.rs

```diff
@@ -933,18 +932,17 @@ fn main() {
 
     // It's unlikely that we are going to cross compile to different OS,
     // but this guarantees that we choose the correct OS APIs if we do so.
-    let target_os_or_current =
-        env::var("CARGO_CFG_TARGET_OS").unwrap_or_else(|_| env::consts::OS.to_string());
+    let target_family_or_current =
+        env::var("CARGO_CFG_TARGET_FAMILY").unwrap_or_else(|_| env::consts::FAMILY.to_string());
 
-    let target_os = match &target_os_or_current as &str {
-        "windows" => OS::Win,
-        "linux" => OS::Linux,
-        "macos" => OS::Darwin,
+    let target_family = match &target_family_or_current as &str {
+        "windows" => OsFamily::Win,
+        "unix" => OsFamily::Unix,
         other => panic!("Unsupported Operational System: {}", other),
```

Comment:
Just a nit: would be nice to change the panic message:

```suggestion
        other => panic!("Unsupported Operational System Family: {}", other),
```